strings_lab.js

```diff
@@ -0,0 +1,255 @@
+// # String Exercises
```

Comment:
Overall good demonstration of string properties and methods. I like the fact that you did all bonus questions as well. Keep up the good work!